packages/aws-cdk/NOTICE

```diff
@@ -1,2 +1,14 @@
 AWS Cloud Development Kit (AWS CDK)
 Copyright 2018-2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
We should have this shorthand in all our notice files, but for now just added it here. 

Still figuring out whether we can add a reference to the `THIRD_PARTY_LICENSES` file here.

Reply:
Added the referral